### PR TITLE
Enhanced: "readme file", "fallback" package naming, Added features: "Custom Fallback Namespace", "Click to Download"

### PR DESCRIPTION
**Pull request by Raruto**
I made the changes based on what was said [here](https://github.com/tomjn/composerpress/commit/252166935173f50278baa0082cbf25482ea779db) and [here](https://github.com/tomjn/composerpress/commit/9a40819c852e8da4c65545138a49e5b2b2450e42#comments).

I have "taken" the functionality of setting a custom vendor name from the [satipress](https://github.com/blazersix/satispress) plugin (@bradyvercher have you ever thought of combining the features offered by these two plugins?)

_I hope_ I have not done too much trouble, this is my first ever Pull Request..

### CODE REVIEW

File: php/toolpage.php [hunk 1]
```diff
@@ -3,26 +3,151 @@
...
+
 	protected $composerpress = null;
 	protected $model = null;
+
 	public function __construct( $composerpress, $model ) {
 		$this->composerpress = $composerpress;
 		$this->model = $model;
+
 		add_action( 'admin_menu', array( $this, 'on_admin_menu' ) );
+    add_action( 'admin_init', array( $this, 'on_admin_init' ) );
```
Comment (tomjn) [inline php/toolpage.php]: Incorrect indenting

**Comment by tomjn** [comment]
Thanks, can you fix the indenting though?

**Comment by Raruto** [comment]
goes better? (not well understood if I did another pull..)

**Comment by tomjn** [comment]
Better, can you fix the double tabs though?
```
{
	//
}
vs
{
		//
}
```

**Comment by Raruto** [comment]
Did you refer to that portion of code?

**Pull request merged**